sdks/wasm/pinvoke-tables-default.h

```diff
@@ -0,0 +1,58 @@
+typedef struct {
```

Comment:
nit i.e. ignorable: Please add a comment saying this is generated, if it is, as I suspect. The generator should output the comment.